event-processing/lambda/tests/unit/services/cleansing-service.test.ts

```diff
@@ -3,29 +3,64 @@ import { CleansingService } from '../../../services/cleansing-service';
 import { IEnrichedAuditEvent } from '../../../models/enriched-audit-event';
 import { ICleansedEvent } from '../../../models/cleansed-event';
 
+const inputMessage: IEnrichedAuditEvent = {
+    timestamp: 1609462861,
+    timestamp_formatted: '2021-01-23T15:43:21.842',
+    event_name: 'AUTHENTICATION_ATTEMPT',
+    event_id: '123456789',
+    component_id: '1234',
+    client_id: 'An Example Client',
+    reIngestCount: 0,
+};
+
 describe('Unit test for cleansing-service', function () {
-    it('returns a cleansed event', async () => {
-        const inputMessage: IEnrichedAuditEvent = {
+
+    afterEach(() => {
```

Comment:
Could we do this in a beforeEach - currently the first test that runs doesn't haven't the user key predefined but it will be for subsequent tests.  This might be confusing if we reorder them later.

Reply:
You could also set the input message to this for the scenario where you don't want the user key on the object at all.

Reply:
I've amended so that user is set on the event const, and then explicitly defined in each test.  Because user is optional and can in theory be undefined, it threw typing errors if trying to just set the user.user_id or user.journey_id properties in each test, this approach works.